python/cm/migrations/0048_auto_20200228_1032.py

```diff
@@ -0,0 +1,96 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Generated by Django 3.0.3 on 2020-02-28 10:32
+# pylint: disable=line-too-long
+
+import json
+import os
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+import cm.config as config
+
+
+def get_body(job, name, type_log, format_log):
+    file_path = os.path.join(config.LOG_DIR, f'{job.id}-{name}-{type_log}.{format_log}')
+    if os.path.exists(file_path):
+        with open(file_path, 'r') as f:
+            body = f.read()
+        return body
+    else:
+        return None
+
+
+def add_logs(apps, schema_editor):
+    LogStorage = apps.get_model('cm', 'LogStorage')
+    JobLog = apps.get_model('cm', 'JobLog')
+    db_alias = schema_editor.connection.alias
+
+    jobs = JobLog.objects.using(db_alias).all()
```

Comment:
using(db_alias) is superfluously.
You can write just JobLog.objects.all()

See, for example https://github.com/arenadata/adcm/blob/develop/python/cm/migrations/0037_auto_20191120_1600.py

Reply:
Ok